tests/test_data/test_monitor_data.py

```diff
@@ -190,8 +186,16 @@ def make_flux_data():
 
 
 def make_directivity_data():
+    data = make_far_field_data_array()
```

Comment:
modify test to include the new fields `left_polarization` and `right_polarization`.